test/cpp/test_lazy.cpp

```diff
@@ -0,0 +1,122 @@
+#include <gtest/gtest.h>
+
+#include "tensorflow/compiler/xla/shape.h"
+#include "torch/csrc/lazy/core/shape.h"
+#include "torch_xla/csrc/helpers.h"
+#include "torch_xla_test.h"
+
+namespace torch_xla {
+namespace cpp_test {
+
+class LazyTest : public TorchXlaTest {};
+
+TEST_F(LazyTest, TestXlaShapeToLazyWithF64) {
```

Comment:
Oops. Thanks for adding more test cases. I didn't mean to ask you to add more tests, just to change a few parameters in the previous test. Hope this didn't create too much overhead for you.